aws-java-sdk-core/src/main/java/com/amazonaws/retry/PredefinedRetryPolicies.java

```diff
@@ -217,7 +221,7 @@ public final long delayBeforeNextRetry(AmazonWebServiceRequest originalRequest,
                 scaleFactor = SCALE_FACTOR;
             }
 
-            long delay = (1 << retriesAttempted) * scaleFactor;
```

Comment:
I'd vote to keep it simple and keep the 1 instead of 1L.  Nothing to do with optimization, but more  to simplicity and readability.